stdlib/src/main/scala/chisel3/std/SBox.scala

```diff
@@ -0,0 +1,344 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  This is translated from https://github.com/riscv/riscv-crypto
+//
+//    2020-01-29    Markku-Juhani O. Saarinen <mjos@pqshield.com>
+//    Copyright (c) 2020, PQShield Ltd. All rights reserved.
+//    Converted to Chisel in 2022
+
+/*
+    [BoPe12] Boyar J., Peralta R. "A Small Depth-16 Circuit for the AES
+    S-Box." Proc.SEC 2012. IFIP AICT 376. Springer, pp. 287-298 (2012)
+    DOI: https://doi.org/10.1007/978-3-642-30436-1_24
+    Preprint: https://eprint.iacr.org/2011/332.pdf
+
+    [Ny93] Nyberg K., "Differentially Uniform Mappings for Cryptography",
+    Proc. EUROCRYPT '93, LNCS 765, Springer, pp. 55-64 (1993)
+    DOI: https://doi.org/10.1007/3-540-48285-7_6
```

Comment:
emmm...
I think the API should be discussed.
Can you join the dev-meeting today or tomorrow to explain the basic idea of your implementation?
I thought it's kind of an affine transformation? I'm not so sure about it.

Reply:
The way to review it would be to copy the results into the test and then run the generator function, checking if they are correct.

Reply:
> The way to review it would be to copy the results into the test and then run the generator function, checking if they are correct.

Sorry I did not quite get it. What "results" and "generator function" did you mean?  I think the current copy-pasted SBox table in the integration test could be "results" and the "generator" is derived by `z3` in SBoxTester, and they are checked by `z3`.